frontend/testing/cypress/src/integration/usecase/usecase.js

```diff
@@ -23,13 +22,8 @@ context(
       });
     });
     beforeEach(() => {
+      cy.studiologin(Cypress.env('useCaseUser'), Cypress.env('useCaseUserPwd'));
       cy.intercept('GET', '**/datamodels').as('getDatamodels');
-      Cypress.Cookies.preserveOnce(
```

Comment:
Nice to get rid of this one, since it was deprecated in V9.7 👏

Reply:
Yes, need to keep stuff updated.